Bottom: 388d7eb9d4d4ef175b26e8451855304858976b40
Top:    8e1cab6864a3540e69e1fe7c92b6cedfcf559148
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-14 16:24:10 +0800

Refresh of vhost-build-xdp

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 6a6ef6a..95aa32d 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -118,7 +118,6 @@ struct vhost_net_virtqueue {
 	struct ptr_ring *rx_ring;
 	struct vhost_net_buf rxq;
 	struct xdp_buff xdp[VHOST_RX_BATCH];
-	struct vring_used_elem heads[VHOST_RX_BATCH];
 };
 
 struct vhost_net {
